src/static/css/2019.css

```diff
@@ -30,6 +30,14 @@ a:focus, a:hover {
   text-decoration: underline;
 }
 
+/* The main wrapper, center aligned */
+.container {
+  max-width: 1280px;
+  padding-left: 20px;
+  padding-right: 20px;
```

Comment:
Any reason we need this padding? Wasn't there previously.

Reply:
When defining a container element, you want to set left and right padding "bleeds" which are both visible and effective on mobile displays. 

Also, I could have used a shorthand like `padding: 0 20px`to get rid of an extra line but this could have caused issues when combining .e.g. with other padding helpers - helpers which we don't have now but we might in the future.

Reply:
Then is it needed on desktop? Just with the reduction from 1324 -> 1200 and then losing another 40px here, we've effectively gone from 1324 to 1160. Not that that is necessarily bad, but thought I'd ask the question.

Reply:
Having set 1280px for the `.container`, the total "remaining" available space for the content is 1240px and not 1160px.